ui/uihandler.go

```diff
@@ -894,15 +894,15 @@ func Report(baseurl, port, staticRsrcDir string, datadir string) {
 // sets the urlHostPort variable.
 func setHostAndPort(inputPort string, addr *net.TCPAddr) {
 	urlHostPort = inputPort
-	portStr := inputPort
+	portStr := ""
```

Comment:
can you explain what this fixes/changes ?

Reply:
If I'm not remembering wrong, addr *net.TCPAddr is not taken care. urlHostPort contains localhost as an address even I gave ip address as an input